src/lib/OpenEXR/ImfStandardAttributes.h

```diff
@@ -140,24 +155,310 @@ IMF_STD_ATTRIBUTE_DEF (longitude, Longitude, float)
 IMF_STD_ATTRIBUTE_DEF (latitude, Latitude, float)
 IMF_STD_ATTRIBUTE_DEF (altitude, Altitude, float)
 
+//
+// cameraMake -- manufacturer or vendor of the camera
+//
+// If present, the value should contain only printable characters
```

Comment:
How about
"If present, the value should contain one or more characters X encoded using Unicode UTF-8 encoding in NFC normal form, as defined in the Unicode standard."

Except leave the "X" out of it. I put it there to be able to say "What's after the 'X' is word-for-word what's in SMPTE ST 2065-4:2023" (and :2013 too, FWIW).

Reply:
Excuse my laziness for not reading all the exr specs and code to answer my own question, but what do we currently assume about the encoding of string attributes? Is there a reason we don't change the spec to just allow/assume them all to be UTF-8?

Reply:
I haven't spotted anything that would break with utf8 character encodings, ie. we don't have any code that would break if the high bit of a character is set. I like "If present, the value should contain one or more characters encoded using Unicode UTF-8 encoding in NFC normal form, as defined in the Unicode standard."

I would be happy to put that at the top of the attributes file as an explanation of what we mean by UTF-8, and just use the word UTF-8 wherever "printable" occurs now. 

Also to Larry's question, yes, we should do that. And we should probably also have some unit tests. But I'd reckon updating the spec and adding tests for utf8 should be a different task than this current one.